lib/winapi/profiling.c

```diff
@@ -3,22 +3,60 @@
 // SPDX-FileCopyrightText: 2019 Stefan Schmidt
 
 #include <profileapi.h>
+#ifdef USE_RDTSC_FOR_FREQ
+#include <synchapi.h>
+#endif
 #include <assert.h>
 #include <stddef.h>
 #include <xboxkrnl/xboxkrnl.h>
 
+#ifdef USE_RDTSC_FOR_FREQ
+static LARGE_INTEGER frequency = {0, 0};
+static void __attribute__((constructor)) PrimeQueryPerformanceFrequency ()
+{
+    #define AVG_SET 2
+    ULARGE_INTEGER f_rdtsc, avg = {0, 0}, s_rdtsc;
```

Comment:
Not sure about C syntax with multiple variables per initializer, but is `f_rdtsc` actually initialized here (it's used before being set below)?

Reply:
Addressed in latest commit.